sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/models/TrainRequest.java

```diff
@@ -7,9 +7,7 @@
 import com.azure.core.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/**
- * The TrainRequest model.
- */
+/** The TrainRequest model. */
```

Comment:
out of curiosity, why all these formatting changes?

Reply:
Yeah, I think autorest for Java underwent some major changes. Talking with them separately on this.